### PR TITLE
Add warning to development docs

### LINKED ISSUE #695: Add warning to docs built from `development`
### Feature Request

Currently we also build documentation for development branch 
https://starknetpy.readthedocs.io/en/development/

We should find a solution of adding note to this docs indicating that these are not for latest stable release.

### PR DESCRIPTION
**Pull request by drknzz**
Closes #695

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -22,7 +22,7 @@
 # -- Project information -----------------------------------------------------
 
 project = "Starknet.py"
-copyright = "2022, Software Mansion"
+copyright = "2023, Software Mansion"
```
Comment (cptartur) [inline docs/conf.py:25]: 👍

File: docs/conf.py [hunk 2]
```diff
@@ -49,6 +49,13 @@
 # This pattern also affects html_static_path and html_extra_path.
 exclude_patterns = ["_build", "Thumbs.db", ".DS_Store"]
 
+# Add note about documentation from development branch
+if os.environ.get("READTHEDOCS_VERSION") == "development":
+    rst_prolog = """.. attention::
+
+        This page was created from `development <https://github.com/software-mansion/starknet.py>`_ branch.
+    """
+
```
Comment (cptartur) [inline docs/conf.py:58]: Looks okay, but was it tested at all?

**Review by war-in — APPROVED** [review]
It can be merged if it was tested 👍

File: docs/conf.py (hunk 2 above)
Comment (drknzz) [reply docs/conf.py:58]: It has been tested in my underground laboratory 🧪

Comment (war-in) [reply docs/conf.py:58]: Could you show us a screenshot? 😏

Comment (drknzz) [reply docs/conf.py:58]: https://starknetpy.readthedocs.io/en/development/

**Pull request merged**